src/views/app/detail-panel/edit/parts/text-editor-container.tsx

```diff
@@ -46,30 +47,22 @@ const TextEditorContainer: FC<PropType> = ({ onDragOverEvent, draftSavedAt, minH
 					crossAlignment="flex-end"
 				>
 					{editor?.richText && composerIsAvailable ? (
-						<Controller
-							height="fit"
-							name="text"
-							control={control}
-							defaultValue={editor?.text}
-							render={({ onChange, value }): ReactElement => (
-								<Container background="gray6" mainAlignment="flex-start" style={{ minHeight }}>
-									<StyledComp.EditorWrapper>
-										<Composer
-											// eslint-disable-next-line @typescript-eslint/ban-ts-comment
-											// @ts-ignore
-											value={value[1]}
-											onEditorChange={(ev: Array<string>): void => {
-												updateSubjectField({ text: [ev[0], ev[1]] });
-												throttledSaveToDraft({ text: [ev[0], ev[1]] });
-												onChange([ev[0], ev[1]]);
-												toggleStickyTime();
-											}}
-											onDragOver={onDragOverEvent}
-										/>
-									</StyledComp.EditorWrapper>
-								</Container>
-							)}
-						/>
+						<Container background="gray6" mainAlignment="flex-start" style={{ minHeight }}>
+							<StyledComp.EditorWrapper>
+								<Composer
+									// eslint-disable-next-line @typescript-eslint/ban-ts-comment
+									// @ts-ignore
+									value={inputValue[1]}
+									onEditorChange={(ev: Array<string>): void => {
+										setInputValue([ev[0], ev[1]]);
+										updateSubjectField({ text: [ev[0], ev[1]] });
+										throttledSaveToDraft({ text: [ev[0], ev[1]] });
```

Comment:
It could be more efficient to create a new array once and using it in the 3 function calls:

```
const values = ev.slice(0, 2);
setInputValue(values);
updateSubjectField({ text: values });
```